fxgl-core/src/test/kotlin/com/almasb/fxgl/core/collection/PropertyMapTest.kt

```diff
@@ -251,6 +253,25 @@ class PropertyMapTest {
         assertTrue(s.contains("testInt=IntegerProperty [value: 3]"))
     }
 
+    private var timesCounter = 0
```

Comment:
Please move `timesCounter`, `sumCounter` and `counter(key: String, value: Any)` to the scope of `forEach test` to be consistent with existing tests where similar approach was used.

Reply:
fixed